sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -546,9 +546,16 @@ object Radar {
         val usage = "initialize"
         this.apiClient.getConfig(usage, false, object : RadarApiClient.RadarGetConfigApiCallback {
             override fun onComplete(status: RadarStatus, config: RadarConfig) {
-                locationManager.updateTrackingFromMeta(config?.meta)
-                RadarSettings.setFeatureSettings(context, config?.meta.featureSettings)
-                RadarSettings.setSdkConfiguration(context, config?.meta.sdkConfiguration)
+                if (status == RadarStatus.SUCCESS) {
```

Comment:
do we also need to include these changes to `sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt`?

Reply:
this is on initialize only, see next PR https://github.com/radarlabs/radar-sdk-android/pull/364